docs/UserGuide.md

```diff
@@ -564,6 +575,28 @@ When editing employee and project IDs directly in the data file, take extra caut
 
 [Return to Top](#table-of-contents)
 
+### Traversing command history
```

Comment:
Could this be moved before "Saving the data file"? I feel it would fit since it is an action the user can perform in app (compared to saving/editing the data file)

Reply:
Done!